.github/workflows/chromatic.yaml

```diff
@@ -5,12 +5,10 @@ name: "chromatic"
 
 on:
   pull_request:
-    types: [labeled, synchronize]
+    types: [labeled]
     branches-ignore:
       - changeset-release/main
-
   push:
```

Comment:
I believe we wanted to have an exemption for `canary` and the changeset branches didn't we?

Reply:
The conditions should prevent that, `canary` doesn't sync `main` and neither does `changeset`

Reply:
I suppose if someone adds `run chromatic` on those branches they messed up anyway...